examples/cfd/stokes_mgn/pi_inference.py

```diff
@@ -0,0 +1,525 @@
+from modulus.sym.manager import GraphManager
+
+from typing import Optional, Dict
+
+
+class Stokes(PDE):
+    """Incompressible Stokes flow"""
+
+    def __init__(self, nu, dim=3):
+        # set params
+        self.dim = dim
+
+        # coordinates
+        x, y, z = Symbol("x"), Symbol("y"), Symbol("z")
+
+        # make input variables
+        input_variables = {"x": x, "y": y, "z": z}
+        if self.dim == 2:
+            input_variables.pop("z")
+
+        # velocity componets
+        u = Function("u")(*input_variables)
+        v = Function("v")(*input_variables)
+        if self.dim == 3:
+            w = Function("w")(*input_variables)
+        else:
+            w = Number(0)
+
+        # pressure
+        p = Function("p")(*input_variables)
+
+        # kinematic viscosity
+        if isinstance(nu, str):
+            nu = Function(nu)(*input_variables)
+        elif isinstance(nu, (float, int)):
+            nu = Number(nu)
+
+        # set equations
+        self.equations = {}
+        self.equations["continuity"] = u.diff(x) + v.diff(y) + w.diff(z)
+        self.equations["momentum_x"] = +p.diff(x) - nu * (
+            u.diff(x).diff(x) + u.diff(y).diff(y) + u.diff(z).diff(z)
+        )
+        self.equations["momentum_y"] = +p.diff(y) - nu * (
+            v.diff(x).diff(x) + v.diff(y).diff(y) + v.diff(z).diff(z)
+        )
+        self.equations["momentum_z"] = +p.diff(z) - nu * (
+            w.diff(x).diff(x) + w.diff(y).diff(y) + w.diff(z).diff(z)
+        )
+
+        if self.dim == 2:
+            self.equations.pop("momentum_z")
+
+
+class DNN(torch.nn.Module):
+    """
+    Custom PyTorch model
+    """
+
+    def __init__(self, layers, fourier_features=64):
+        super(DNN, self).__init__()
+
+        # parameters
+        self.depth = len(layers) - 1
+
+        # Fourier features
+        self.fourier_features = fourier_features
+        self.B = 10 * torch.randn((layers[0], fourier_features)).to(
+            "cuda"
+        )  # Random matrix
+
+        # set up layer order dict
+        self.activation = torch.nn.GELU
+
+        layer_list = list()
+        for i in range(1, self.depth - 1):
+            layer_list.append(
+                ("layer_%d" % i, torch.nn.Linear(layers[i], layers[i + 1]))
+            )
+            layer_list.append(("activation_%d" % i, self.activation()))
+
+        layer_list.append(
+            ("layer_%d" % (self.depth - 1), torch.nn.Linear(layers[-2], layers[-1]))
+        )
+        layerDict = OrderedDict(layer_list)
+
+        # deploy layers
+        self.layers = torch.nn.Sequential(layerDict)
+
+    def forward(self, x):
+        # Add Fourier features
+        x_proj = torch.matmul(x, self.B)
+        x_proj = torch.cat([torch.sin(x_proj), torch.cos(x_proj)], dim=-1)
+
+        # Pass through layers
+        out = self.layers(x_proj)
+        return out
+
+
+class MdlsSymDNN(Arch):
+    """
+    Wrapper model to convert PyTorch model to Modulus-Sym model.
```

Comment:
Add some more docs here with links to what a Key is, and how this wrapper does the mapping

Reply:
Thanks, added!